ci/travis-install-conda.sh

```diff
@@ -5,6 +5,6 @@ bash ~/miniconda.sh -b -p ~/miniconda
 source ~/miniconda/bin/activate
 
 conda install --yes --quiet python=3.7
-conda install --yes --quiet tensorflow=${conda_tf_version}
-
+conda install --yes --quiet nomkl tensorflow=${conda_tf_version}
+conda install --yes --quiet numpy=1.18.5
```

Comment:
Drop a comment why this is needed; also while you're here, add a comment also why `nomkl` was needed

Reply:
Prefereably add to this comment a diff b/w transitive deps of newer vs older numpy

Reply:
In case we ever face a similar issue in the future :/

Reply:
Ok, but it works without specifying `nomkl` so I'm leaving that out. Supposedly all thanks to correct BLAS version